src/incubator/calendar/types.ts

```diff
@@ -129,7 +129,7 @@ export interface CalendarProps {
 export interface AgendaProps {
   renderEvent?: (event: Event) => React.ReactElement | null;
   renderHeader?: (header: DateSectionHeader) => React.ReactElement | null;
-  itemHeight?: number;
+  itemHeight?: number | null;
```

Comment:
Why adding null and not removing the itemHeight default value if we want to let it take each item's height? Does it has to do with the scrollToIndex?

Reply:
I decided to remove this prop entirely (I also found a TODO for that), we don't seem to use or need it really